Makefile

```
@@ -115,6 +109,28 @@ $(GOBIN)/labeler:
 label: $(GOBIN)/labeler
 	labeler apply misc/triage/labels.yaml -r aquasecurity/trivy-operator -l 5
 
+## Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
+## controller-gen version and binary path
+CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
+CONTROLLER_TOOLS_VERSION ?= v0.9.1
```

```suggestion
CONTROLLER_TOOLS_VERSION ?= v0.9.2
```

New patched released today. 😄

---

updated the version